src/styles/tailwind/list.ts

```diff
@@ -0,0 +1,17 @@
+import { tv, type VariantProps } from 'tailwind-variants';
+
+export const listVariants = tv({
+	base: 'rounded-lg p-4',
+	variants: {
+		variant: {
+			pink: 'bg-[#FFEBE7]',
+			grey: 'bg-[#F7F7F7]',
+			default: 'bg-white'
+		}
+	},
```

Comment:
I think @punchanabu provide the preset color already, you can see in the `tailwind.config.ts` or `color.ts`
So you can easily use that custom color
e.g.
```ts
variant: {
    pink: 'bg-sucu-pink-05',
    grey: 'bg-sucu-gray-light',
    default: 'bg-white'
}
```
<img width="111" alt="Screenshot 2024-09-14 at 2 42 42 AM" src="https://github.com/user-attachments/assets/9d9230e1-da13-4f5e-866a-e1523953a489">

which also match in Figma design

Reply:
It's just a minor change krub but overall is a great!
Good Job!👏